README.md

```diff
@@ -2,25 +2,43 @@
 
 > IPA narrow transcription of English words in New York City accent
 
-## Demo Website
+## Online Demo Website
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/6cc76b39-d72c-4d05-9959-0d57d6b8b6e4/deploy-status)](https://app.netlify.com/sites/ipa-jfk/deploys)
 
 You can lookup words on our hassle-free **[Demo Website](https://jfk.b1f6c1c4.info/)**!
 
-## Install
+## Prerequisites
+
+- Node.js 12.x+
+- Yarn (recommended)
+- [Parcel](https://parceljs.org/getting_started.html) (for demo)
```

Comment:
You should point to `https://v2.parceljs.org/`

Reply:
Will do

Reply:
But beware of the migration tips [here](https://v2.parceljs.org/getting-started/migration/).